features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/conf_templates/templates/repository/conf/api-manager.xml.j2

```diff
@@ -146,6 +143,26 @@
         </Environments>
     </APIGateway>
 
+    <TokenIssuers>
+    {%- for issuer_1 in apim.jwt.issuer -%}
+    <TokenIssuer issuer ="{{issuer_1.name}}">
+    {% if issuer_1.jwks is defined %}
+      <JWKSConfiguration>
+        <URL>{{issuer_1.jwks.url}}</URL>
```

Comment:
is jwks always open?

Reply:
for most of the vendors endpoint was open